starter-code/src/functions-and-arrays.js

```diff
@@ -33,6 +76,18 @@ var wordsArr = [
   'palace'
 ];
 
+function averageWordLength (wordsArr) {
+  var allLetters = [0];
+  for (var k = 0; k < wordsArr.length; k++) {
+    allLetters += wordsArr[k].length;
+  }
+  if (allLetters == 0) {
```

Comment:
You should check this in the beginning of the function and better with something like `wordsArr.length > 0`